mlte/frontend/nuxt-app/pages/index.vue

```diff
@@ -158,25 +163,21 @@ const path = ref([
   },
 ]);
 
-const namespaces = ref(["Default", "TEST 1", "Super longg purposes"]);
-
-const selectedNamespace = ref("Default");
+const { data: namespaceOptions } = await useFetch<string[]>(
+  "http://localhost:8080/api/namespace",
```

Comment:
Not necessary to update this for this PR, but we will need to make this configurable to alllow users to run the store in locations other than `localhost:8080`. We can default to some value like this in development mode, but when run from the CLI, we should accept an argument that is the address of the artifact store.

Reply:
I created an issue for this.

Reply:
Yes, still want to get that implemented. I am not sure that there is going to be a simple way to make that happen so I just wanted to get this along with the rest of the frontend implemented before going down that rabbit hole.